drivers/synopsys/ufs/dw_ufs.c

```diff
@@ -97,10 +97,20 @@ static int dwufs_phy_set_pwr_mode(ufs_params_t *params)
 	int result;
 	unsigned int data, tx_lanes, rx_lanes;
 	uintptr_t base;
+	int flags;
 
 	assert((params != NULL) && (params->reg_base != 0));
 
 	base = params->reg_base;
+	flags = params->flags;
+
+	if(flags & UFS_FLAGS_VENDOR_SKHYNIX) {
```

Comment:
Style check error - add space between if and ( - if (flags...).

Static analysis error - flags & UFS_FLAGS_VENDOR_SKHYNIX is *not* boolean; should be:

if ((flags & UFS_FLAGS_VENDOR_SKHYNIX) != 0)